glass/fields.py

```diff
@@ -250,7 +245,7 @@ def generate_gaussian(
         ncorr = ngrf - 1
 
     # number of modes
-    n = max((len(gl) for gl in gls if gl is not None), default=0)  # type: ignore[redundant-expr]
+    n = max((len(gl) for gl in gls), default=0)
```

Comment:
`None` entries should be supported for backward compatibility

Reply:
As discussed, I'm deprecating the `None` option. But have restored the type as `int | tuple[int, ...]`